dockercloud/http.go

```diff
@@ -5,10 +5,13 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/http/cookiejar"
 	"os"
 )
 
 var customUserAgent = "go-dockercloud/" + version
+var client *http.Client
+var jar http.CookieJar
```

Comment:
yep, I pushed again and put the exact same implementation as the event container.